spec/api/v2/schemas/security_guides.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require './spec/api/v2/schemas/util'
```

Comment:
If you do the recursive require in the correct order above, you might not need this.